library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapter.java

```diff
@@ -793,6 +793,122 @@ void callSuperNotifyItemMoved(int fromPosition, int toPosition) {
         super.notifyItemMoved(fromPosition, toPosition);
     }
 
+    /**
+     * Helper method that calls {@link #notifyItemChanged} when section changed from
+     * LOADING/ FAILED/ EMPTY to LOADING/ FAILED/ EMPTY.
```

Comment:
Please change the doc here and the others to:
> Helper method that calls {@link #notifyItemChanged} with the position of the {@link State} view holder in the adapter.
> Useful to be called after changing the State from LOADING/FAILED/EMPTY to LOADING/FAILED/EMPTY

Sorry for being picky, the reason is that "... when ..." implies that something is being done "automatically" as if there were listeners to some events